### PR TITLE
chore(ci): Switch from coveralls to codecov

### PR DESCRIPTION
**Pull request by janmeier**

### CODE REVIEW

**Comment by felixfbecker** [comment]
You forgot the readme badge ;)

File: package.json [hunk 1]
```diff
@@ -59,9 +59,9 @@
     "chai-as-promised": "^5.1.0",
     "chai-datetime": "^1.4.0",
     "chai-spies": "^0.7.0",
+    "codecov": "^1.0.1",
```
Comment (felixfbecker) [inline package.json]: Can remove this dep if we do it with bash

File: package.json [hunk 2]
```diff
@@ -130,7 +130,7 @@
     "cover-integration": "cross-env COVERAGE=true node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -t 60000 --ui tdd \"test/integration/**/*.test.js\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/integration.info')\"",
     "cover-unit": "cross-env COVERAGE=true node_modules/.bin/istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -t 30000 --ui tdd \"test/unit/**/*.test.js\" && node -e \"require('fs').renameSync('coverage/lcov.info', 'coverage/unit.info')\"",
     "merge-coverage": "lcov-result-merger \"coverage/*.info\" \"coverage/lcov.info\"",
-    "coveralls": "npm run cover && cat ./coverage/lcov.info | node ./node_modules/coveralls/bin/coveralls.js && rimraf ./coverage",
+    "ci-coverage": "npm run cover && codecov",
```
Comment (felixfbecker) [inline package.json]: Can remove this script aswell

File: appveyor.yml [hunk 3]
```diff
@@ -34,10 +31,7 @@ before_test:
...
-
-notifications:
-  - provider: Webhook
-    url: https://coveralls.io/webhook?repo_token=751SiA6QhY5EH4P2ygq7JFWulqiD2ETXO
-    on_build_success: true
-    on_build_failure: true
+  - 'IF "%COVERAGE%" == "true" (npm run cover) ELSE (npm test)'
+  
+after_test:
+  - ps: (Invoke-WebRequest https://codecov.io/bash).toString() | bash -f .\coverage\lcov.info
```
Comment (felixfbecker) [inline appveyor.yml]: Should be
```yaml
 - ps: if (Test-Path env:\COVERAGE) { (Invoke-WebRequest https://codecov.io/bash).toString() | bash -f .\coverage\lcov.info }
```
Might wrap it multiple lines:
```yaml
 - ps: |
   if (Test-Path env:\COVERAGE) {
     (Invoke-WebRequest https://codecov.io/bash).toString() | bash -f .\coverage\lcov.info
   }
```

Comment (janmeier) [inline appveyor.yml]: Seems it's interpreting the coverage file as bash

image

**Comment by janmeier** [comment]
Sweet mother of jebus, it works! Thanks for the powershell help @felixfbecker !

**Pull request merged**